packages/raja/package.py

```diff
@@ -161,10 +161,10 @@ class Raja(CachedCMakePackage, CudaPackage, ROCmPackage):
     variant("openmp", default=False, description="Build OpenMP backend")
     variant("shared", default=False, description="Build shared libs")
     variant("desul", default=False, description="Build desul atomics backend")
-    variant("vectorization", default=False, description="Build SIMD/SIMT intrinsics support")
-    variant(
-        "omptask", default=False, description="Build OpenMP task variants of internal algorithms"
-    )
+    variant("vectorization", default=True, description="Build SIMD/SIMT intrinsics support")
+    variant("omptask", default=False, description="Build OpenMP task variants of internal algorithms")
+    variant("omptarget", default=False, description="Build OpenMP on target device support")
+    variant("sycl", default=False, description="Build sycl backend")
```

Comment:
@long58 @rhornung67 what are the minimum required versions to activate omptarget and/or sycl in RAJA ?
I would like to add the corresponding conflicts.
Also, are omptarget and sycl mutually exclusive ? If so, I’d like to add that conflict too.

Reply:
OpenMP target and SYCL are mutually exclusive.

I don't know the minimum versions of camp and RAJA to support OpenMP target. That has been supported in both for years.

There is no minimum released version of RAJA to support SYCL. I recently merged a PR to develop for this to support all RAJA SYCL tests that are enabled and also the kernels I have been adding to RAJA Perf. So, the next RAJA release would be the minimum release tag.